evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -316,6 +351,20 @@ def add_result(self, score_to_minimize, pipeline, trained_pipeline_results):
                     "RF Classifier Select From Model"
                 ).get_names()
 
+            try:
+                ohe = pipeline.get_component("One Hot Encoder")
```

Comment:
Rather than the try except can we just check if the One Hot Encoder component exists in the pipeline?

I think since this lives in the default algorithm, and the default algorithm has full control over the pipelines it creates (unlike iterative algo), I think it's ok since the One Hot Encoder will always have the same name?